visualizer.js

```diff
@@ -218,6 +214,9 @@ function onSample (sample) {
             offsetForGrid(sample.channelData[index], index)
         );
         channel.shift();
+        channel = notchFilter.multiStep(channel);
+        channel = lpFilter.multiStep(channel);
+        channel = hpFilter.multiStep(channel);
```

Comment:
it looks like each channel is being filtered on every sample in this code. I don't think this is how we should implement the time series filter.  My recommendation would be to remove the time series filter code from this PR -- but leave the notch filter for the FFT -- and then merge the PR.  Then, we'll reformulate the time series code with the filters implemented in a new PR once we've had a chance to meet an figure out a game plan